docs/rest-api/rest-notebook.md

```diff
@@ -433,6 +434,58 @@ limitations under the License.
   <table class="table-configuration">
     <col width="200">
     <tr>
+      <th>Run paragraph synchronously</th>
+      <th></th>
+    </tr>
+    <tr>
+      <td>Description</td>
+      <td>This ```POST``` method runs the paragraph synchronously by given notebook and paragraph id.
+      This API can return SUCCESS or ERROR depending on the outcome of the paragraph execution
+      </td>
+    </tr>
+    <tr>
+      <td>URL</td>
+      <td>```http://[zeppelin-server]:[zeppelin-port]/api/notebook/run/[notebookId]/[paragraphId]```</td>
+    </tr>
+    <tr>
+      <td>Success code</td>
+      <td>200</td>
+    </tr>
+    <tr>
+      <td> Fail code</td>
```

Comment:
thanks, I'm referring to whitespaces in/around `<td> Fail code</td>`, `<td> 500 </td>` or `<td> sample JSON input (optional, only needed when if you want to update dynamic form's value) </td>` - are they intentional?